vision/cloud-client/src/main/java/com/example/vision/SetEndpoint.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.vision;
+
+import com.google.cloud.vision.v1.AnnotateImageRequest;
+import com.google.cloud.vision.v1.AnnotateImageResponse;
+import com.google.cloud.vision.v1.BatchAnnotateImagesResponse;
+import com.google.cloud.vision.v1.EntityAnnotation;
+import com.google.cloud.vision.v1.Feature;
+import com.google.cloud.vision.v1.Image;
+import com.google.cloud.vision.v1.ImageAnnotatorClient;
+import com.google.cloud.vision.v1.ImageAnnotatorSettings;
+import com.google.cloud.vision.v1.ImageSource;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+class SetEndpoint {
+
+  // Change your endpoint
+  static void setEndpoint() throws IOException {
+    // [START vision_set_endpoint]
+    ImageAnnotatorSettings settings =
+        ImageAnnotatorSettings.newBuilder().setEndpoint("eu-vision.googleapis.com:443").build();
+
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    ImageAnnotatorClient client = ImageAnnotatorClient.create(settings);
```

Comment:
Nice - Is the client thread safe?  (ie can it be used concurrently?)

Reply:
Normally I'd put it in the `try` resource block, but because the product team didn't want anything else on the docs page for this snippet, the region tag ends right below this.
But I still wanted to make sure this was testable, so the rest of the snippet runs the code then closes the client, but the actual snippet is only the above 2 lines on the docs.